js-exercises/update-object/updateObject.js

```diff
@@ -0,0 +1,15 @@
+function updateObject(...args) {
+	const position = args[0];
+	const item = args[1];
+	const array = args[2];
+
+	if (args.length < 3) return 'Missing arguments. Expected: index(number), item(any data type), array(array)';
```

Comment:
Also, generally many prefer to have, 'if statements' to span across multiple lines and have brackets!